src/lib/wicked/adapter.js

```diff
@@ -96,7 +96,10 @@ class WickedAdapter {
     onInterfaceChange(fn) {
         this.client.onInterfaceChange((signal, iface) => {
             const data = (signal === 'deviceDelete') ? { ...iface, link: false } : iface;
-            fn(signal, createInterface(data));
+            // Do not trust in the calculated managed value. When changing an interface
+            // status the `client_state` element might be missing.
+            const { managed, ...newIface } = createInterface(data);
```

Comment:
NP: I have no strong opinion on this, but maybe we can use the _underscore connvention_ and put `managed: _managed`

Reply:
Good point. The next time I will change this code I will do it :)